emf/src/test/java/io/lionweb/lioncore/java/emf/EMFMetamodelExporterTest.java

```diff
@@ -136,4 +147,152 @@ public void exportInterfaceAndEnumeration() {
+
+    EClass propertiesFile = (EClass) propertiesPkg.getEClassifier("PropertiesFile");
+    assertEquals(Arrays.asList(eObject, iNamed), propertiesFile.getESuperTypes());
+    assertEquals("PropertiesFile", propertiesFile.getName());
+    assertFalse(propertiesFile.isAbstract());
+    assertFalse(propertiesFile.isInterface());
+    assertEquals(1, propertiesFile.getEStructuralFeatures().size());
+
+    EClass property = (EClass) propertiesPkg.getEClassifier("Property");
+    assertEquals(Arrays.asList(eObject, iNamed), property.getESuperTypes());
+    assertEquals("Property", property.getName());
+    assertFalse(property.isAbstract());
+    assertFalse(property.isInterface());
+    assertEquals(1, property.getEStructuralFeatures().size());
+
+    EClass value = (EClass) propertiesPkg.getEClassifier("Value");
+    assertEquals(Arrays.asList(eObject), value.getESuperTypes());
+    assertEquals("Value", value.getName());
+    assertFalse(value.isAbstract());
+    assertFalse(value.isInterface());
+    assertEquals(0, value.getEStructuralFeatures().size());
+
+    EClass booleanValue = (EClass) propertiesPkg.getEClassifier("BooleanValue");
+    assertEquals(Arrays.asList(value), booleanValue.getESuperTypes());
+    assertEquals("BooleanValue", booleanValue.getName());
+    assertFalse(booleanValue.isAbstract());
+    assertFalse(booleanValue.isInterface());
+    assertEquals(1, booleanValue.getEStructuralFeatures().size());
+
+    EClass decValue = (EClass) propertiesPkg.getEClassifier("DecValue");
+    assertEquals(Arrays.asList(value), decValue.getESuperTypes());
+    assertEquals("DecValue", decValue.getName());
+    assertFalse(decValue.isAbstract());
+    assertFalse(decValue.isInterface());
+    assertEquals(1, decValue.getEStructuralFeatures().size());
+
+    EClass intValue = (EClass) propertiesPkg.getEClassifier("IntValue");
+    assertEquals(Arrays.asList(value), intValue.getESuperTypes());
+    assertEquals("IntValue", intValue.getName());
+    assertFalse(intValue.isAbstract());
+    assertFalse(intValue.isInterface());
+    assertEquals(1, intValue.getEStructuralFeatures().size());
+
+    EClass stringValue = (EClass) propertiesPkg.getEClassifier("StringValue");
+    assertEquals(Arrays.asList(value), stringValue.getESuperTypes());
+    assertEquals("StringValue", stringValue.getName());
+    assertFalse(stringValue.isAbstract());
+    assertFalse(stringValue.isInterface());
+    assertEquals(1, stringValue.getEStructuralFeatures().size());
+
+    EReference props = (EReference) propertiesFile.getEStructuralFeature("props");
+    assertEquals("props", props.getName());
+    assertEquals(1, props.getLowerBound());
+    assertEquals(-1, props.getUpperBound());
+    assertEquals(property, props.getEType());
+
+    EReference valueRef = (EReference) property.getEStructuralFeature("value");
+    assertEquals("value", valueRef.getName());
+    assertEquals(1, valueRef.getLowerBound());
+    // TODO fix
```

Comment:
What is the issue here?

Reply:
setting the upper bound to -1 is the workaround for #111